tests/core/factory/test_validation_definition_factory.py

```diff
@@ -308,6 +310,53 @@ def _test_validation_definition_factory_delete_success(
         context.validation_definitions.get(name)
 
 
+@pytest.mark.filesystem
+def test_validation_definition_factory_round_trip(
+    empty_data_context: FileDataContext,
+    validation_definition: ValidationDefinition,
+):
+    # Arrange
+    context = empty_data_context
+
+    ds = context.sources.add_pandas("my_ds")
+    csv_path = (
+        pathlib.Path(__file__).parent.parent.parent
+        / "test_sets"
+        / "quickstart"
+        / "yellow_tripdata_sample_2022-01.csv"
```

Comment:
```suggestion
        pathlib.Path(__file__, "..", "..", "..", "test_sets", "quickstart" ,"yellow_tripdata_sample_2022-01.csv").resolve()
```
totally optional, but you could also do this.